### PR TITLE
copyrightnotice added to metadata

### PR DESCRIPTION
**Pull request by vied12**

### CODE REVIEW

File: superdesk/metadata/item.py [hunk 1]
```diff
@@ -246,6 +246,14 @@
         'type': 'string',
         'nullable': True
     },
+    'copyrightnotice': {
+        'type': 'string',
+        'nullable': True
+    },
+    'usageterms': {
```
Comment (mdhaman) [inline superdesk/metadata/item.py]: `usageterms` already exists in the `item.py`

**Comment by vied12** [comment]
Thank you @mdhaman. I removed it and moved `copyrightnotice` next to the existing `usageterms`

**Comment by ahilles107** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**